content/en/docs/ops/ambient/architecture/index.md

```diff
@@ -13,4 +13,77 @@ This page is under construction.
 
 ## Traffic routing
 
+In ambient mode, workloads can fall into 3 categories:
+* Uncaptured: this is a standard pod without any mesh features enabled.
+* Captured: this is a pod that has traffic intercepted by `ztunnel`. Pods can be captured by setting the `istio.io/dataplane-mode=ambient` label on a namespace.
+* Waypoint enabled: this is a pod that is "Captured" *and* has a waypoint proxy deployed.
+  A waypoint will, by default, apply to all pods in the same namespace.
+  It can optionally be set to apply to only a specific service account with the `istio.io/for-service-account` annotation on the `Gateway`.
+  If there is both a namespace waypoint and service account waypoint, the service account waypoint takes precedence.
+
+Depending on which category a workload is in, the request path will be different.
+
+### Ztunnel routing
+
+#### Outbound
+
+When a captured pod makes an outbound request, it will be transparently redirected to `ztunnel` which will determine where and how to forward the request.
+In general, the traffic routing behaves just like Kubernetes default traffic routing;
+requests to a `Service` will be sent to an endpoint within the `Service` while requests directly to a `Pod` IP will go directly to that IP.
+
+However, depending on the destination's capabilities, different behavior will occur.
+If the destination is also captured, or otherwise has Istio proxy capabilities (such as a sidecar), the request will be upgraded to an encrypted HBONE tunnel.
+If the destination has a waypoint proxy, in addition to being upgraded to HBONE, the request will instead be forwarded to that waypoint.
+
+Note that in the case of a request to a `Service`, a specific endpoint will be selected to determine if it has a waypoint.
```

Comment:
Is it possible to explain how the specific endpoint would be selected?  Is it randomly?

Reply:
That is an implementation detail - currently its random but we will optimize it in the future. I don't mind putting it but it will change

Reply:
I see, ok, SGTM not documenting it for now.